data/json/items/gun/50.json

```diff
@@ -67,7 +67,9 @@
     },
     "material": [ "steel", "wood" ],
     "clip_size": 1,
+    "modes": [],
```

Comment:
Yes that is indeed better
@Rivet-the-Zombie feel free to update the wiki documentation I wrote for that as we should aim for consistency.